Improve readme (#937)
LLNL/RAJA: README.md

[![Join the chat at https://gitter.im/llnl/raja](https://badges.gitter.im/llnl/raja.svg)](https://gitter.im/llnl/raja?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
[![Coverage](https://img.shields.io/codecov/c/github/LLNL/RAJA/develop.svg)](https://codecov.io/gh/LLNL/RAJA)

RAJA is a library of C++ software abstractions, primarily developed at

> Lawrence Livermore National Laboratory (LLNL), that enable architecture

Reviewer: ```suggestion
Lawrence Livermore National Laboratory (LLNL), that enables architecture
```

Author: Done.

and programming model portability for HPC applications. RAJA has two main goals:

  * To enable application portability with manageable disruption to existing 
    algorithms and programming styles
  * To achieve performance comparable to using common programming models, 
    such as OpenMP, CUDA, etc. directly.

> RAJA targets portable, parallel loop execution by providing building blocks 

Reviewer: ```suggestion
RAJA offers portable, parallel loop execution by providing building blocks 
```

Author: Done.

that extend the generally-accepted parallel for idiom. RAJA relies on standard 
C++11 features, such as lambda expressions.

> RAJA is rooted in a perspective based on decades of experience working on 

Reviewer: ```suggestion
RAJA's design is rooted in decades of experience working on 
```

Author: Done.

> RAJA's desing is rooted in decades of experience working on 

Reviewer 2: design

Author: Fixed. Thanks.

production mesh-based multiphysics applications. Based on the diversity of
algorithms and software engineering styles used in such applications, RAJA 
is designed to enable application developers to adapt RAJA concepts and 
specialize them for different code implementation patterns and C++ usage.

RAJA shares goals and concepts found in
other C++ portability abstraction approaches, such as
[Kokkos](https://github.com/kokkos/kokkos)
and [Thrust](https://developer.nvidia.com/thrust). 
However, it includes concepts and capabilities that are absent in other models 

> and which are fundamental to applications we work with.

Reviewer 2: remove either 'and' or 'which' here

Author: Done.
